src/services/UserService.ts

```
@@ -0,0 +1,22 @@
+    return res.json();
+  }
+
+  static async getUserExperience(userId: string, token: string) {
+    const res = await fetch(`${this.BASE_URL}/api/experiences/${userId}`, {
+      headers: { Authorization: `Bearer ${token}` },
+    });
+
+    if (!res.ok) throw new Error(`Failed to fetch experience: ${res.status}`);
+    return res.json();
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)